include/d/d_camera.h

```diff
@@ -86,25 +113,33 @@ class dCamera_c {
     /* 0x03C */ cSGlobe m03C;
     /* 0x044 */ cXyz m044;
     /* 0x050 */ cXyz m050;
-    /* 0x05C */ s16 m05C;
+    /* 0x05C */ cSAngle m05C;
     /* 0x05E */ u8 m05E[0x060 - 0x05E];
     /* 0x060 */ f32 m060;
-    /* 0x064 */ u8 m064[0x068 - 0x064];
+    /* 0x064 */ f32 m064;
     /* 0x068 */ int m068;
     /* 0x06C */ cSAngle mAngleY;
-    /* 0x06E */ u8 m06E[0x080 - 0x06E];
+    /* 0x06E */ u8 m06E[0x070 - 0x06E];
+    /* 0x070 */ cXyz m070;
+    /* 0x07C */ u32 m07C;
     /* 0x080 */ u32 m080;
     /* 0x084 */ cXyz m084;
     /* 0x090 */ cXyz m090;
     /* 0x09C */ f32 m09C;
-    /* 0x0A0 */ f32 m0A0;
+    /* 0x0A0 */ cSAngle m0A0;
+    /* 0x0A2 */ u8 m0A2[0x0A4 - 0x0A2];
     // /* 0x0A4 */ @class$4017d_camera_cpp m0A4[2];
```

Comment:
I think this is an anonymous struct? Take a look at some of the structs in d_vibration.h for an example. Dunno what goes in the struct though

Reply:
I added a struct which seemed to work but not sure how to resolve the name (if that even matters).

Reply:
By "anonymous" I mean that it doesn't have any name specified by the programmer, that's why it gets a compiler-generated name:
```suggestion
    /* 0x0A4 */ struct {
        /* 0x00 */ int m00;
        /* 0x04 */ int m04;
        /* 0x08 */ int m08;
        /* 0x0C */ int m0C;
        /* 0x10 */ int m10;
        /* 0x14 */ int m14;
        /* 0x18 */ int m18;
        /* 0x1C */ int m1C;
    } m0A4[2];
```
I'm not sure if it would get the exact same name ("d_camera_cpp" seems to imply it's defined in the cpp file...? that doesn't seem right) but I guess this would be close enough

Reply:
You can't give it an explicit ctor/dtor, but the compiler will create one automatically if the struct has at least one member with a non-trivial ctor of its own (like an array or a cXyz). It looks like there might actually be nested anonymous structs here (`dCamera_c::@class$4336d_camera_cpp::@class$4337d_camera_cpp::~@class$4337d_camera_cpp()`) meaning the inner anonymous struct has at least one member with a ctor, and then the outer struct might inherit the ctor from the inner one.